include/pmix_common.h.in

```diff
@@ -604,6 +604,8 @@ typedef int pmix_status_t;
 #define PMIX_GDS_ACTION_COMPLETE                (PMIX_ERR_OP_BASE - 18)
 /* gap created by v3 definitions */
 #define PMIX_OPERATION_SUCCEEDED                (PMIX_ERR_OP_BASE - 27)
+/* gap for group codes */
+#define PMIX_PROC_TERMINATED                    (PMIX_ERR_OP_BASE - 38)
```

Comment:
We should update the standard doc with `PMIX_PROC_TERMINATED`

Reply:
Yeah, I pinged you about that yesterday but you were busy. It's obviously something we missed and should have had.

Reply:
Sorry I didn't see the ping. Apparently I've not bee getting those notifications - I'm just seeing them now :(